### PR TITLE
Fix some issues in the analyticstack docs

### PR DESCRIPTION
**Pull request by mulby**
A few users reported problems while installing the analyticstack. Hopefully these changes will help clarify things!
### Reviewers
- [x] Subject matter expert: @brianhw 
- [ ] Subject matter expert: @HassanJaveed84 
- [x] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc

FYI: @cpennington @thallada @dsjen 
### Testing
- [ ] Ran ./run_tests.sh without warnings or errors

### CODE REVIEW

File: en_us/install_operations/source/installation/analytics/start_analytics.rst [hunk 1]
```diff
@@ -132,6 +130,13 @@ Starting the Open edX Analytics Pipeline
            --marker hdfs://localhost:9000/edx-analytics-pipeline/output/answer_distribution_raw/$UNIQUE_NAME/marker \
            --n-reduce-tasks 1
 
+#. Run other tasks.
```
Comment (lamagnifica) [inline en_us/install_operations/source/installation/analytics/start_analytics.rst]: maybe be a little more specific about the purpose of these other tasks? or do you expect this to grow over time, so you're minimizing expected rework in the future?

File: en_us/install_operations/source/installation/analytics/start_analytics.rst [hunk 2]
```diff
@@ -100,30 +100,28 @@ Starting the Open edX Analytics Pipeline
 
 #. Navigate to the course body and submit answers to a few problems.
 
-#. Navigate to the location where edx-analytics-pipeline project was cloned on
```
Comment (lamagnifica) [inline en_us/install_operations/source/installation/analytics/start_analytics.rst:103]: is this definitely no longer useful to orient the people following these steps? we do get occasional questions about "where I'm supposed to be".

**Comment by lamagnifica** [comment]
@mulby a couple of suggestions to consider. 👍

File: en_us/install_operations/source/installation/analytics/start_analytics.rst (hunk 1 above)
Comment (mulby) [reply en_us/install_operations/source/installation/analytics/start_analytics.rst]: Developers will add new tasks... so I was trying to provide them with a tiny template for how they might run new stuff they built. Maybe I should just add a section on creating a new task?

Comment (mulby) [reply en_us/install_operations/source/installation/analytics/start_analytics.rst]: @cpennington what do you think?

File: en_us/install_operations/source/installation/analytics/start_analytics.rst (hunk 2 above)
Comment (mulby) [reply en_us/install_operations/source/installation/analytics/start_analytics.rst:103]: This instruction is now actually incorrect, so it had to go!

We don't want them to do anything on the host machine anymore...

**Comment by mulby** [comment]
@cpennington @dsjen @lamagnifica - I added a section here about creating a task.

It feels more like developer docs... not sure exactly where to put it.

**Comment by brianhw** [comment]
LGTM.  👍

**Comment by mulby** [comment]
@edx/doc I changed this pretty significantly since the last review, can one of you take another pass over it?

**Comment by mulby** [comment]
@HassanJaveed84 can you review this as well?

**Comment by srpearce** [comment]
@lamagnifica Do you have time to do another once-over on this? If not, just let @catong or me know.

**Comment by catong** [comment]
@srpearce Can you please review this PR?

File: en_us/install_operations/source/installation/analytics/example_create_task.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+.. contents::
+   :local:
+   :depth: 1
+
+*******************
+Example: Word Count
+*******************
+
+#. Create a new python module for your task.
```
Comment (srpearce) [inline en_us/install_operations/source/installation/analytics/example_create_task.rst]: Nit: Capitalize Python

File: en_us/install_operations/source/installation/analytics/example_create_task.rst [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+#. Add the following line to the ``edx.analytics.tasks`` section of ``setup.cfg``:
+
+   .. code-block:: text
+
+     word-count = edx.analytics.tasks.word_count:WordCountTask
+
+   .. note::
+
+        Modifying setup.cfg is only necessary when you are adding a new python
```
Comment (srpearce) [inline en_us/install_operations/source/installation/analytics/example_create_task.rst]: Nit: Capitalize Python

**Review by srpearce — APPROVED** [review]
Two very small nits.

**Comment by mulby** [comment]
@srpearce do you mind re-publishing this guide when you get a chance?

**Pull request merged**